server/auth/local.js

```diff
@@ -11,7 +11,7 @@ router.post('/signup', async (req, res, next) => {
   }
 });
 
-router.get('/login', async (req, res, next) => {
+router.put('/login', async (req, res, next) => {
   try {
```

Comment:
Why did we change this to a put request? 
Should login change anything on the User table row? 
I looked at the login method and I don't see anything going on there...

Reply:
I changed it to a put so I can send along some data and configure the request.

Reply:
If you can do the same with a get, I will change it back.